oe_theme.theme

```diff
@@ -418,6 +420,16 @@ function oe_theme_preprocess_pattern_dialog(array &$variables): void {
   $variables['hidden'] = $variables['hidden'] ? 'true' : 'false';
 }
 
+/**
+ * Implements hook_preprocess_HOOK() for pattern gallery template.
```

Comment:
We are just using ` * Implements hook_preprocess_HOOK().` on all other hooks, let's keep it this way for consistency.

Reply:
:+1: